### PR TITLE
Eh 743 nayton jarjestaja enrichening

### PR DESCRIPTION
**Pull request by vesaruu**
Get organization data for naytonJarjestaja using oppilaitosOid

### CODE REVIEW

File: shared/models/NaytonJarjestaja.ts [hunk 1]
```diff
@@ -1,6 +1,15 @@
 import { types } from "mobx-state-tree"
+import { Organisaatio } from "./Organisaatio"
+import { EnrichOrganisaatioOid } from "./EnrichOrganisaatioOid"
 
-export const NaytonJarjestaja = types.model("NaytonJarjestaja", {
+const Model = types.model("NaytonJarjestaja", {
```
Comment (J-Kallunki) [inline shared/models/NaytonJarjestaja.ts]: Is this `"NaytonJarjestaja", ` required here?

File: shared/models/helpers/TutkinnonOsa.ts [hunk 2]
```diff
@@ -29,38 +30,13 @@ export interface MuuOppimisymparisto {
...
-  loppu: string
-  jarjestaja?: { oppilaitosOid?: string }
-  nayttoymparisto: Nayttoymparisto
-  tyoelamaOsaamisenArvioijat: TyoelamaOsaamisenArvioija[]
-  koulutuksenJarjestajaOsaamisenArvioijat: KoulutuksenJarjestajaOsaamisenArvioija[]
-  sisallonKuvaus: string[]
-  yksilollisetKriteerit: string[]
-  vaatimuksistaTaiTavoitteistaPoikkeaminen: string
-}
+export interface OsaamisenOsoittaminen
```
Comment (J-Kallunki) [inline shared/models/helpers/TutkinnonOsa.ts]: maybe `IOsaamisenOsoittaminen` ?

File: shared/models/NaytonJarjestaja.ts (hunk 1 above)
Comment (vesaruu) [reply shared/models/NaytonJarjestaja.ts]: True, not needed. Forgot to remove it when I created composed model.

File: shared/models/helpers/TutkinnonOsa.ts (hunk 2 above)
Comment (vesaruu) [reply shared/models/helpers/TutkinnonOsa.ts]: Yes, I'll change. Forgot to add that prefix in previous pr also, I'll change that too.

File: shared/models/NaytonJarjestaja.ts (hunk 1 above)
Comment (vesaruu) [reply shared/models/NaytonJarjestaja.ts]: Now that I checked, it seems that we have named compose model "root"-models with "Model" postfix. MST documentation says that models should have name for debugging purposes.

**Pull request merged**